### DISCUSSION TITLE
some errors appears

### DISCUSSION THREAD

**Discussion by ronbere** [vladmandic/sdnext#1140] [Q&A]
23:32:12-187264 ERROR    Module load: D:\automatic\extensions-builtin\LDSR\scripts\ldsr_model.py: ModuleNotFoundError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ D:\automatic\modules\script_loading.py:10 in load_module                                                            │
│                                                                                                                     │
│    9 │   try:                                                                                                       │
│ > 10 │   │   module_spec.loader.exec_module(module)                                                                 │
│   11 │   except Exception as e:                                                                                     │
│ in exec_module:883                                                                                                  │
│ in _call_with_frames_removed:241                                                                                    │
│                                                                                                                     │
│ D:\automatic\extensions-builtin\LDSR\scripts\ldsr_model.py:8 in <module>                                            │
│                                                                                                                     │
│    7 from modules.upscaler import Upscaler, UpscalerData                                                            │
│ >  8 from ldsr_model_arch import LDSR                                                                               │
│    9 from modules import shared, script_callbacks                                                                   │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
ModuleNotFoundError: No module named 'ldsr_model_arch'
23:32:12-565623 ERROR    Module load: D:\automatic\extensions-builtin\ScuNET\scripts\scunet_model.py:
                         ModuleNotFoundError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ D:\automatic\modules\script_loading.py:10 in load_module                                                            │
│                                                                                                                     │
│    9 │   try:                                                                                                       │
│ > 10 │   │   module_spec.loader.exec_module(module)                                                                 │
│   11 │   except Exception as e:                                                                                     │
│ in exec_module:883                                                                                                  │
│ in _call_with_frames_removed:241                                                                                    │
│                                                                                                                     │
│ D:\automatic\extensions-builtin\ScuNET\scripts\scunet_model.py:14 in <module>                                       │
│                                                                                                                     │
│    13 from modules import devices, modelloader                                                                      │
│ >  14 from scunet_model_arch import SCUNet as net                                                                   │
│    15 from modules.shared import opts                                                                               │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
ModuleNotFoundError: No module named 'scunet_model_arch'
ControlNet v1.1.189
ControlNet v1.1.189
23:32:17-607798 ERROR    Module load: D:\automatic\extensions-builtin\SwinIR\scripts\swinir_model.py:
                         ModuleNotFoundError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ D:\automatic\modules\script_loading.py:10 in load_module                                                            │
│                                                                                                                     │
│    9 │   try:                                                                                                       │
│ > 10 │   │   module_spec.loader.exec_module(module)                                                                 │
│   11 │   except Exception as e:                                                                                     │
│ in exec_module:883                                                                                                  │
│ in _call_with_frames_removed:241                                                                                    │
│                                                                                                                     │
│ D:\automatic\extensions-builtin\SwinIR\scripts\swinir_model.py:11 in <module>                                       │
│                                                                                                                     │
│    10 from modules.shared import opts, state                                                                        │
│ >  11 from swinir_model_arch import SwinIR as net                                                                   │
│    12 from swinir_model_arch_v2 import Swin2SR as net2                                                              │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
ModuleNotFoundError: No module named 'swinir_model_arch'
[-] ADetailer initialized. version: 23.5.17, num models: 7
[AddNet] Updating model hashes...
100%|██████████████████████████████████████████████████████████████████████████████| 106/106 [00:00<00:00, 4276.49it/s]
[AddNet] Updating model hashes...
100%|█████████████████████████████████████████████████████████████████████████████| 106/106 [00:00<00:00, 52947.03it/s]
23:32:19-507054 INFO     Loading UI theme: gradio/base
Create LRU cache (max_size=16) for preprocessor results.
Create LRU cache (max_size=16) for preprocessor results.
Running on local URL:  http://127.0.0.1:7860

Windows10 , Chrome last commit

**Comment by vladmandic** [comment]
you have no description - is this a new install? upgrade? did it ever work? etc...
and why is this a discussion topic?

**Reply by ronbere** [reply]
because I don't want to clutter the issues so I thought I'd post here
yes, I reinstalled everything from the beginning. I just have these errors

**Reply by vladmandic** [reply]
This looks like something bad happened during actual install, now you're just looking at consequences.

**Reply by ronbere** [reply]
Should I erase everything again, and redo a new installation, possibly on another hard drive?

**Reply by vladmandic** [reply]
I would.

**Reply by ronbere** [reply]
Hi Vladimir, fresh install on another disk, I get this on launch:

Traceback (most recent call last):
  File "D:\seait_installers_version_0.1.4\automatic\venv\lib\site-packages\gradio\routes.py", line 422, in run_predict
    output = await app.get_blocks().process_api(
  File "D:\seait_installers_version_0.1.4\automatic\venv\lib\site-packages\gradio\blocks.py", line 1323, in process_api
    result = await self.call_function(
  File "D:\seait_installers_version_0.1.4\automatic\venv\lib\site-packages\gradio\blocks.py", line 1051, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "D:\seait_installers_version_0.1.4\automatic\venv\lib\site-packages\anyio\to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "D:\seait_installers_version_0.1.4\automatic\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
asyncio.exceptions.CancelledError
13:56:38-316729 WARNING  WebSocket closed (ignore asyncio.exceptions.CancelledError)



Windows10, chrome

**Reply by vladmandic** [reply]
i don't think you can get this on launch, you get this on restart from ui ? if yes, see what it it says in the last line? those are open connections from browser to server that get killed when you restart server and if i hide those, i might be hiding something important as well.

**Reply by ronbere** [reply]
you're right, now after the last commit, have many issues, would you than i post here or on the issues section?
Sorry my english are so bad

**Reply by vladmandic** [reply]
Create an issue.
